framework/src/headers/mod.rs

```diff
@@ -3,21 +3,28 @@ pub use self::mac::*;
 pub use self::null_header::*;
 pub use self::tcp::*;
 pub use self::udp::*;
+use std::net::Ipv6Addr;
+
 pub mod ip;
 pub mod mac;
-
 mod null_header;
 mod tcp;
 mod udp;
 
-// L4 Protocol Next Header Values
+// L3/L4 Protocol Next Header Values
 pub const TCP_NXT_HDR: u8 = 6;
 pub const UDP_NXT_HDR: u8 = 17;
-pub const ICMP_NXT_HDR: u8 = 1;
```

Comment:
is the long term goal to support both ipv4 and ipv6 traffic ? i believe these consts were being used for ipv4

Reply:
yeah. ipv4 still works fine. this nxt_hdr stuff is v6 only anyhow, but placed in mod.rs here b/c of where the other protocols exist.